net/sslh/src/opnsense/mvc/app/controllers/OPNsense/Sslh/SettingsController.php

```diff
@@ -0,0 +1,64 @@
+<?php
+
+/*
+    Copyright (C) 2018 Michael Muenz <m.muenz@gmail.com>
```

Comment:
you might want to add your own, I don't see a reason to split hairs on copyright for a file with 2 lines of "code"